app/mailers/event/leader_reminder_mailer.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+class Event::LeaderReminderMailer < ApplicationMailer
+  include Rails.application.routes.url_helpers
+
+  REMINDER = "event_leader_reminder"
+
+  def reminder(course)
+    return if course.contact.nil?
+
+    @course = course
+    values = values_for_placeholders(REMINDER)
+    headers = {bcc: course.groups.first.course_admin_email}
```

Comment:
Is here save access needed? Or can't `groups` be empty?

```suggestion
    headers = {bcc: course.groups.first&.course_admin_email}
```

Reply:
the course url is `/de/groups/1/events/1`, so groups.first cannot be empty